diff --git a/configure b/configure
index 3b2636035..a74b88b41 100755
--- a/configure
+++ b/configure
@@ -1470,6 +1470,8 @@ fi
 # Determine if system stack overflows can be detected
 
 case "$arch,$system" in
+  *)
+    inf "System stack overflow detechion DISABLED.";;
   i386,linux_elf|amd64,linux|power,rhapsody|amd64,macosx|i386,macosx \
   |amd64,openbsd|i386,bsd_elf)
     inf "System stack overflow can be detected."
